sos/report/plugins/maas.py

```diff
@@ -8,133 +8,168 @@
-                "/var/snap/maas/common/snap_mode",
-                "/var/snap/maas/current/*.conf",
-                "/var/snap/maas/current/bind",
-                "/var/snap/maas/current/http",
-                "/var/snap/maas/current/supervisord",
-                "/var/snap/maas/current/preseeds",
-                "/var/snap/maas/current/proxy",
-                "/var/snap/maas/current/syslog",
+                    "/var/snap/maas/common/log",
+                    "/var/snap/maas/current/supervisord",
             ])
         else:
             self.add_copy_spec([
-                "/etc/squid-deb-proxy",
-                "/etc/maas",
-                "/var/lib/maas/dhcp*",
-                "/var/lib/maas/http/*.conf",
-                "/var/lib/maas/*.conf",
-                "/var/lib/maas/rsyslog",
-                "/var/log/maas*",
-                "/var/log/upstart/maas-*",
-            ])
-            self.add_cmd_output([
-                "apt-cache policy maas-*",
-                "apt-cache policy python-django-*",
-            ])
-
-        if self.is_installed("maas-region-controller"):
-            self.add_cmd_output([
-                "maas-region dumpdata",
+                    "/var/snap/maas/common/log/*.log",
+                    "/var/snap/maas/current/supervisord/*.log",
+                ] + (self._get_machines_most_recent_syslog(
+                        "/var/snap/maas/common/log/rsyslog"
+                    )
+                )
+            )
+
+    def _deb_collect(self):
+        self.add_cmd_output([
+            "apt-cache policy maas maas-*",
+        ])
+
+        self.add_forbidden_path([
+            "/var/lib/maas/**/*.key",
+            "/var/lib/maas/**/*.pem",
+            "/var/lib/maas/**/secret",
+            "/etc/maas/**/*.key",
+            "/etc/maas/**/*.pem",
+            "/etc/maas/**/secret",
+        ])
+
+        self.add_copy_spec([
+            "/etc/maas/**/*.conf",
+            "/etc/maas/**/*.yaml",
+            "/etc/maas/preseeds",
+            "/var/lib/maas/**/*.conf",
+            "/var/lib/maas/dhcp/*.leases",
+            "/var/lib/maas/temporal",
+        ])
+
+        if self.get_option("all_logs"):
+            self.add_copy_spec([
+                    "/var/log/maas/",
             ])
+        else:
+            self.add_copy_spec([
+                    "/var/log/maas/**/*.log",
+                ] + (self._get_machines_most_recent_syslog(
+                        "/var/log/maas/rsyslog"
+                    )
+                )
+            )
 
-        if self._has_login_options():
-            if self._remote_api_login():
-                self.add_cmd_output(f"maas {self.get_option('profile-name')} "
-                                    "commissioning-results list")
-            else:
-                self._log_error(
-                    "Cannot login into MAAS remote API with provided creds.")
+    def setup(self):
+        for service in self._services:
```

Comment:
@arif-ali I decided not to use `services` because I don't want default behaviour as defined in https://github.com/sosreport/sos/blob/ca7300e7007c8b4faf7502ec3afb3a891267c2b2/sos/report/plugins/__init__.py#L3334-L3347

I would like to call `add_journal` with `since -1days` by default

Reply:
yeah, I think that's what was here before from the MAAS plugin